utils/s2n_random.c

```diff
@@ -420,15 +429,15 @@ int s2n_get_rdrand_data(struct s2n_blob *out)
             }
         }
 
-        S2N_ERROR_IF(!success, S2N_ERR_RDRAND_FAILED);
+        ENSURE(success, S2N_ERR_RDRAND_FAILED);
 
         int data_to_fill = MIN(sizeof(output), space_remaining);
 
-        GUARD(s2n_stuffer_write_bytes(&stuffer, output.u8, data_to_fill));
+        GUARD_AS_RESULT(s2n_stuffer_write_bytes(&stuffer, output.u8, data_to_fill));
     }
 
-    return 0;
+    return S2N_RESULT_OK;
 #else
-    S2N_ERROR(S2N_ERR_UNSUPPORTED_CPU);
+    BAIL(S2N_ERR_UNSUPPORTED_CPU);
```

Comment:
Are we OK with this? Or should it be `S2N_BAIL`

Reply:
I wanted to be consistent with `GUARD` and not prefix it. The same question goes for `ENSURE` and friends.

Since they're all private macros I think we're ok to go without the prefix.